app/components/tree_link_component/tree_link_component.html.haml

```diff
@@ -6,7 +6,9 @@
     %a{id: @child.id, data: @data, title:  @tooltip,
               href: @href, class: "tree-link #{@muted_style} #{@active_style} #{border_left} #{open?}"}
       = @pref_label_html
-
+      - if is_reused
+        .tree-view-reuse-icon
```

Comment:
this need `class` needs to be in the tree view style file.

Reply:
We don't have a specific css file for the tree view, and the other classes are in bioportal.scss file which already has 1248 lines of code

Reply:
Yes it is not good thing, here create a tree view style file, and put it there, and try maybe also to move the other classes related to tree_view there 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/60695275-ca94-4207-8567-690acc2f805e)